releases/latest/mysql-bundle.yaml

```diff
@@ -3,20 +3,31 @@ applications:
     channel: latest/edge
     charm: mysql
     constraints: arch=amd64
-    num_units: 1
-    revision: 80
+    num_units: 3
+    revision: 81
     to:
-    - '0'
+      - "0"
```

Comment:
these need to be single quoted (as the script that autogenerates this uses single quotes). otherwise, a new PR will be auto-opened with single quotes here

Reply:
roger